lib/workers/branch/get-updated.ts

```diff
@@ -213,21 +213,24 @@ export async function getUpdatedPackageFiles(
       }
     }
   }
-  const updatedPackageFiles = Object.keys(updatedFileContents).map((name) => ({
-    name,
+  const updatedPackageFiles: FileAddition[] = Object.keys(
```

Comment:
Are we sure we only have additions here?

Reply:
They were constructed as additions, by setting `contents` fields from `updatedFileContents[name]`, not the `name` itself (as for deletions). I believe we're doint only package files additions, not deletions. For artifact upgrades, we have `File[]` signature couple lines below: this is when all deletions are accumulated, if I my understanding is correct.

Reply:
Yes that's true, currently we only have deletions from artifacts